script.js

```diff
@@ -0,0 +1,129 @@
+// Brand Currency form -------------
+let brandObjectHistory = [{}];
+let eurRateMoving = 0;
+let usdRateMoving = 0;
+let gpyRateMoving = 0;
+
+document
+  .getElementById("brand-currency-form")
+  .addEventListener("submit", brandCurrency);
+
+function brandCurrency(event) {
+  event.preventDefault();
+
+  let baseCurrency = document.getElementById("base-currency").value;
+  let eurRate = document.getElementById("eur").value;
+  let usdRate = document.getElementById("usd").value;
+  let gpyRate = document.getElementById("gpy").value;
+
+  const brandObject = {
+    timestamp: Date.now(),
+    base: baseCurrency,
+    date: new Date().toLocaleDateString(),
+    rate: {
+      eurRate: eurRate,
+      usdRate: usdRate,
+      gpyRate: gpyRate,
+    },
+  };
+
+  brandObjectHistory.push(brandObject);
```

Comment:
Thank you.
I have changed this one from array to object:
 let brandObjectHistory = {};
 But I am confused with the way that you have used that :)
 brandObjectHistory[baseCurrency] = brandObject
 
Because I want to get access to the 'key', which is "base" here and you have assigned the object to the 'baseCurrency'!!
Sorry, I am totally confused with that :).

Reply:
no worries @katayouny , I was a bit confused by the assignment as well. I will check it out now and let you know, but hey no worries. You are on a good track!!

Reply:
@katayouny Also could you please push the changes to github? I cannot see them

Reply:
Hi 
[@Petr](https://hackyourfuture-cph.slack.com/team/U05KA0GN40G)

Thank you for your comments again. I have just received your messages/comments. I don't know what has happened. I am working on my code and commenting to you, but I did not send a pull request:face_in_clouds:. I will send you a new pull request soon.:blush: